### ISSUE TITLE
Conform to modern a11y standards

### ISSUE THREAD

**Issue by Vessel9817** [Vessel9817/onion-site-template#170]
Accessibility is an important part of the web. Much like how all buildings should have a wheelchair-accessible entrance, every website should have a11y features.

**Comment by Vessel9817** [comment]
Relevant web files:
- [src/express/views/**/*](https://github.com/Vessel9817/onion-site-template/blob/5c6f6e515fbbb94350a9c9640fa919eaf1a2e0b8/src/express/views)
- [src/nginx/html/error.html](https://github.com/Vessel9817/onion-site-template/blob/5c6f6e515fbbb94350a9c9640fa919eaf1a2e0b8/src/nginx/html/error.html)
